Tools/Algorithms/input_file_parser.py

```diff
@@ -0,0 +1,32 @@
+def parse_input_file(input_file):
+    """
+    Parse WarpX input file.
+
+    Parameters
+    ----------
+    input_file : string
+        Path to input file.
+
+    Returns
+    -------
+    input_dict : dictionary
+        Dictionary storing WarpX input parameters
+        (parameter's name stored as key, parameter's value stored as value).
+    """
+    input_dict = dict()
+    with open(input_file) as ff:
+        for line in ff:
+            sline = line.split('=')
+            # skip lines that are commented out, blank, or continuation of previous parameters
+            skip_line = sline[0].startswith('#') or sline[0].startswith('\n') or len(sline) == 1
```

Comment:
This is good enough for now. The format of the input file is much more flexible though, for example allowing multiple parameters in a line, though those things probably are not used often.